src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
+export const addBook = (book) => ({
+  type: ADD,
+  payload: book,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE,
+  id,
+});
+
+const reducerBooks = (state = [], action) => {
+  switch (action.type) {
+    case ADD:
+      return [
+        ...state, action.payload];
+    case REMOVE:
+      return state.filter((item) => item !== action.payload);
```

Comment:
- [ ] Let's try to be consistent with our code. In `removeBook` action creator above what you have is `id`, not `pyaload`. so kindly change this to `action.id`, instead of `action.payload`.